vars/perfRegressionBareMetalPipeline.groovy

```diff
@@ -0,0 +1,323 @@
+#!groovy
+import groovy.json.JsonSlurper
```

Comment:
@k0machi 

one last thing, can you change code to run with `perfRegressionParallelPipeline.groovy`, it's now should have everything you were missing, i.e. creating sct runner

Reply:
The parallel pipeline will try to deploy extra instances for read, write and mixed tests, where as this pipeline only does specified subtest in parameters. Since we only have one set of baremetal instances I think that's not going to work.

Reply:
it's depend on the list of subtest you supply to it, some tests have only one, and should work in you case.

I rather not duplicate pipelines, unless there's a real reason, so far I'm no convinced there is